### ISSUE TITLE
First PWA launched results in black screen

### ISSUE THREAD

**Issue by AMoonRabbit** [filips123/PWAsForFirefox#339]
OS: Fedora 38 Workstation
Kern: 6.2.15-300.fc38.x86_64
DE: Gnome 44.1
Browser: Firefox 113.0.1
Ext Ver: 2.6.0
PWAS: 2 installed

Issue:
After a reboot, or log out and log in, the first attempt at starting a PWA will result in a black screen and will go no further. Starting a second, separate PWA will work however, and can be recreated with both installed PWA's after a fresh log in. If the black screen is closed, and re-attempt opening the same PWA a few moments after, the PWA will work.

PWAs are installed with their own profiles.

**Comment by filips123** [comment]
I tried to reproduce this, but it works fine for me...

Please run your web app from command line, so you can obtain logs:

```bash
firefoxpwa profile list # To get the ID of your web app
firefoxpwa site launch YOUR-ID -- --jsconsole # To launch web app with console enabled
```

Then, a console log window should open. Please share logs that you see there.

Another thing you can try is to go to the extension settings and reinstall the runtime.

**Comment by AMoonRabbit** [comment]
I tried the command, but it returns `[ERROR] Web app does not exist`
I tried another one as well
```
firefoxpwa profile list                                      
========================= Default ==========================
Description: Default profile for all web apps
ID: 00000000000000000000000000

=================== Eientei Social MkIII ===================
Description: * Nothing *
ID: 01H0R2XYH1BWVR0YPHKWRKCPXB

Apps:
- Eientei Social MkIII: https://expunged/manifest.json (01H0R2XYQ3T23QG5PTHR8DSRFJ)

====================== Eientei Groove ======================
Description: * Nothing *
ID: 01H0R3REGZA5C39PPD18YD111H

Apps:
- Eientei Groove: https://expunged/manifest.json (01H0R3REW9Z8TNTPMMY3TBGRFP)
```

And then trying to start either results in the following output:
```
firefoxpwa site launch 01H0R3REGZA5C39PPD18YD111H -- --jsconsole
20:59:44 [ERROR] Web app does not exist
```

**Comment by AMoonRabbit** [comment]
Okay, I carried out the reinstall of the runtime from the extension menu, but also played with other settings (disabled the active URLS option) and found that both PWAs just opened a new tab of Firefox instead of what they should be. Removed both PWAs, and set them up as new again and so far, they are both working first time after a fresh user session and reboot. So this is resolved. I will update if issues re-appear. Thank you for taking the time to review my issue.
